pagerduty/resource_pagerduty_business_service_test.go

```diff
@@ -71,6 +71,8 @@ func TestAccPagerDutyBusinessService_Basic(t *testing.T) {
 						"pagerduty_business_service.foo", "point_of_contact", pointOfContact),
 					resource.TestCheckResourceAttrSet(
 						"pagerduty_business_service.foo", "self"),
+					resource.TestCheckResourceAttr(
+						"pagerduty_business_service.foo", "dependency_type", pointOfContact),
```

Comment:
I think this should have been:

```suggestion
						"pagerduty_business_service.foo", "dependency_type", "business_service"),
```